Use dotenv to set up environment variables

alexagranov opened the pull request:
In preparation for adding some new features, I'd like to be able to run tests against my own credentials.  Pulling in https://github.com/bkeepers/dotenv will make it easier to swap out credentials in an `.env` file.

Should we perhaps rename to `.env.default` and require people running tests to either copy the file to `.env` or provide their own credentials?  This would of course go in the Readme.

szich commented:
This is a great addition, thanks for taking the time to do it.

I like your idea of setting up the .env.default

In addition, I think It would be nice to have an .env.test file so the tests would run 'out of the box'.

What are your thoughts?

alexagranov commented:
Hi @szich -

I've added the `.env.default` as well as moved the credentials into `.env.test`.  

Upon first run of `rake test` it will copy `.env.test` over to `.env` and Dotenv will load as usual.  Subsequent runs will respect the presence of `.env`.

I've then added `.env` to `.gitignore` so that developers can modify `.env` with their own credentials without fear of them getting accidentally committed to the repo.

I've also added a section to the README titled "Running Tests".

Let me know what you think.

szich commented:
Looks great, thanks!

alexagranov commented:
Thanks for the quick merge!
